build.xml

```diff
@@ -313,19 +355,34 @@
     </if>
 
     <!-- Run tests -->
-    <if>
-      <istrue value="${phpunit_remote_coverage}" />
-      <then>
-        <property name="paramCoverageDir" value="VUFIND_REMOTE_COVERAGE_DIR=${coveragedir}/tmp" />
-      </then>
-      <else>
-        <property name="paramCoverageDir" value="" />
-      </else>
-    </if>
-    <exec dir="${srcdir}/module/VuFind/tests" command="XDEBUG_MODE=coverage VUFIND_MINK_DRIVER=${mink_driver} VUFIND_SCREENSHOT_DIR=${screenshot_dir} ${phpunit_html_validation_params} VUFIND_SELENIUM_BROWSER=${selenium_browser} VUFIND_SNOOZE_MULTIPLIER=${snooze_multiplier} VUFIND_DEFAULT_TEST_TIMEOUT=${default_test_timeout} VUFIND_LOCAL_DIR=${localdir} VUFIND_URL=${vufindurl} VUFIND_TEST_THEME=${test_theme} SOLR_PORT=${solr_port} ${paramCoverageDir} ${phpunit_command} --log-junit ${builddir}/reports/phpunit.xml --coverage-php ${coveragedir}/tmp/unit.cov ${phpunit_extra_params}" passthru="true" checkreturn="true" />
+    <exec dir="${srcdir}/module/VuFind/tests" executable="${phpunit_command}" passthru="true" checkreturn="true">
+      <arg line="--log-junit ${builddir}/reports/phpunit.xml --coverage-php ${coveragedir}/tmp/unit.cov ${phpunit_extra_params}" />
+      <env key="XDEBUG_MODE" value="coverage" />
+      <env key="VUFIND_MINK_DRIVER" value="${mink_driver}" />
```

Comment:
Feels like it might be beneficial at some point to refactor so we don't have to copy and paste all these variables to so many different places, but that's a project for another day... just thinking out loud!

Reply:
I agree, but it also complicates things. I wish there was support for a set of env vars...